### ISSUE TITLE
how to share sails.js environment with server side workers and daemons

### ISSUE THREAD

**Issue by kkotlarski** [balderdashy/sails#3354]
Hi how can I share sails.js environment (models, services, sockets etc.) with node scripts executed on the server? I've seen scheduling tasks with node-schedule but I'm looking for something more flexible and possibly running in separate process. For example sails.js based API + workers reusing config, models and services to push more date to API database.

**Comment by sailsbot** [comment]
Thanks for posting, @kkotlarski.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jamesearl** [comment]
:+1: would love an answer to this as well.
